README.md

```diff
@@ -1094,6 +1095,89 @@ Server: spray-can/1.3.3
 }
 ```
 
+## GET /workflows/:version/:id/call-metadata
+
+This endpoint returns a superset of the data from #get-workflowsversionidlogs in essentially the same format.
+In addition to stdout and stderr this also returns inputs, start and end date, backend-specific job id, and
+return code.
+  
+### Notes
+
+- The logs endpoints could possibly be deleted once this is implemented, nobody is calling those anyway (but
+  check with Chet first).
+  
+- It seems strange that callers want inputs to calls but not outputs (other than stdout and stderr).
+
+- We never actually specify the backend on which the call executed.  One might be able to infer that from
```

Comment:
I don't think we even capture that. This is a good point, I'll leave it up to you if you think we should add that, no one has asked for it yet so in true scrum fashion we should let it be - but it seems like it'd be useful